net.sourceforge.jpdftweak.jPdfTweak.json

```diff
@@ -0,0 +1,65 @@
+{
+  "app-id" : "net.sourceforge.jpdftweak.jPdfTweak",
+  "branch" : "stable",
+  "runtime" : "org.freedesktop.Platform",
+  "runtime-version" : "19.08",
+  "sdk" : "org.freedesktop.Sdk",
+  "sdk-extensions" : [ "org.freedesktop.Sdk.Extension.openjdk11" ],
+  "command": "net.sourceforge.jpdftweak.jPdfTweak",
+  "finish-args" : [
+    "--env=PATH=/app/jre/bin:/usr/bin:/app/bin",
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+	"--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--filesystem=host",
+    "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
```

Comment:
I believe you don't need this under 19.08, I may be incorrect though

Reply:
I removed this part too. And changed the filesystem to `--filesystem=home`.